velodyne_driver/src/lib/input.cc

```diff
@@ -90,55 +97,57 @@ namespace velodyne_driver
    *  @param private_nh ROS private handle for calling node.
    *  @param port UDP port number
    */
-  InputSocket::InputSocket(ros::NodeHandle private_nh, uint16_t port):
+  InputSocket::InputSocket(rclcpp::Node * private_nh, uint16_t port):
     Input(private_nh, port)
   {
     sockfd_ = -1;
-    
-    if (!devip_str_.empty()) {
-      inet_aton(devip_str_.c_str(),&devip_);
-    }    
+
+    if (!devip_str_.empty())
+      {
+        ::inet_aton(devip_str_.c_str(), &devip_);
+      }
 
     // connect to Velodyne UDP port
-    ROS_INFO_STREAM("Opening UDP socket: port " << port);
-    sockfd_ = socket(PF_INET, SOCK_DGRAM, 0);
+    RCLCPP_INFO(private_nh->get_logger(), "Opening UDP socket: port %u", port);
+
+    sockfd_ = ::socket(PF_INET, SOCK_DGRAM, 0);
     if (sockfd_ == -1)
       {
-        perror("socket");               // TODO: ROS_ERROR errno
+        RCLCPP_ERROR(private_nh->get_logger(), "Error opening socket: %s", ::strerror(errno));
         return;
       }
-  
+
     sockaddr_in my_addr;                     // my address information
     memset(&my_addr, 0, sizeof(my_addr));    // initialize to zeros
     my_addr.sin_family = AF_INET;            // host byte order
-    my_addr.sin_port = htons(port);          // port in network byte order
+    my_addr.sin_port = ::htons(port);          // port in network byte order
     my_addr.sin_addr.s_addr = INADDR_ANY;    // automatically fill in my IP
-  
-    if (bind(sockfd_, (sockaddr *)&my_addr, sizeof(sockaddr)) == -1)
+
+    if (::bind(sockfd_, (sockaddr *)&my_addr, sizeof(sockaddr)) == -1)
       {
-        perror("bind");                 // TODO: ROS_ERROR errno
+        RCLCPP_ERROR(private_nh->get_logger(), "Error binding to socket: %s", ::strerror(errno));
         return;
       }
-  
-    if (fcntl(sockfd_,F_SETFL, O_NONBLOCK|FASYNC) < 0)
+
+    if (::fcntl(sockfd_, F_SETFL, O_NONBLOCK|FASYNC) < 0)
       {
-        perror("non-block");
+        RCLCPP_ERROR(private_nh->get_logger(), "Error settign socket to non-blocking: %s", ::strerror(errno));
```

Comment:
Typo, "setting"

Reply:
Eek.  Good call; fixed in a0752c1a1d87792a5da43945d76caba3fe3b0742